.github/workflows/build-electron.yml

```diff
@@ -50,14 +50,12 @@ jobs:
       run: npm run test-flask-app
 
     - name: Test electron app with puppeteer
-      uses: GabrielBB/xvfb-action@v1
       if: matrix.os != 'macos-latest'
-      continue-on-error: true
```

Comment:
Whoops, I added this recently, but I didn't realize it would give the success indicator on error. I thought it would show the failure but continue anyway. Anyway, thanks for changing it back!

Reply:
Yeah, I thought that's what it'd be doing as well!  Alas, it silently continued